NEWS

```diff
@@ -1,3 +1,28 @@
+11-June-2018 IGNITION v0.26.0
+
+  Features:
+
+    - Expand partitioning to support partition matching, specifying
+      partitions should not exist, and creating existing partitions
```

Comment:
`specifying that a partition should not exist`

Could also drop `Expand partitioning to`.

`creating existing partitions` reads oddly; `creating partitions that may already exist`?

Reply:
whoops, was supposed to be "recreating existing partitions"